src/Expander.hs

```diff
@@ -386,8 +398,57 @@ initializeKernel = do
   traverse_ (uncurry addExprPrimitive) exprPrims
   traverse_ (uncurry addModulePrimitive) modPrims
   traverse_ (uncurry addDeclPrimitive) declPrims
+  traverse_ (uncurry addTypePrimitive) typePrims
 
   where
+    typePrims :: [(Text, SplitTypePtr -> Syntax -> Expand ())]
+    typePrims =
+      [ ( "Bool"
+        , \ dest stx -> do
+            _actualName <- mustBeIdent stx
+            linkType dest TBool
```

Comment:
Looks like the body of this lambda is just copy/pasted with different `T*` constructors, maybe add a `let` binding for it before this list?

Reply:
Good idea!

Reply:
Done!